tests/installation/select_patterns.pm

```diff
@@ -107,6 +102,11 @@ sub run {
             };
             assert_screen 'current-pattern-selected', 5;
         }
+
+        # stick to the default patterns
+        if (get_var('PATTERNS', '') =~ /default/) {
+            $needs_to_be_selected = $selected;
```

Comment:
actually you broke the complete logic now as you add another if between the if and the elsif